test/test_contrib.py

```diff
@@ -368,3 +368,35 @@ def test_exception(self, device, dtype):
         stitcher = kornia.contrib.ImageStitcher(matcher).to(device=device, dtype=dtype)
         with pytest.raises(RuntimeError):
             stitcher(input1, input2)
+
+
+class TestHistMatch:
+
+    def test_interp(self, device, dtype):
+        xp = torch.tensor([1, 2, 3], device=device, dtype=dtype)
+        fp = torch.tensor([4, 2, 0], device=device, dtype=dtype)
+        x = torch.tensor([4, 5, 6], device=device, dtype=dtype)
+        x_hat_expected = torch.tensor([-2., -4., -6.], device=device, dtype=dtype)
+        x_hat = kornia.contrib.interp(x, xp, fp)
+        assert (x_hat_expected == x_hat).all()
+
+    def test_histmatch(self, device, dtype):
+        torch.manual_seed(44)
+        src = torch.randn(1, 4, 4, device=device, dtype=dtype)
+        dst = torch.randn(1, 16, 16, device=device, dtype=dtype)
```

Comment:
```suggestion
        # generate random value by CPU.
        src = torch.randn(1, 4, 4).to(device=device, dtype=dtype)
        dst = torch.randn(1, 16, 16).to(device=device, dtype=dtype)
```

Reply:
@edgarriba Can you commit this? It somehow claims that I do not have write permission to commit the code changes. Any ideas?

Reply:
What it differs from the other?